.github/workflows/ci.yml

```diff
@@ -0,0 +1,152 @@
+name: Build & Test
+on: [pull_request]
```

Comment:
this wont be run when its pushed to master. Do we want to run all sims when it's pushed to master as well?

Reply:
How would it be execute when pushed to master? Does it run in the background or something? What does the SDK do?

Reply:
currently, all repos run all ci when anything is pushed/pullrequested. so when a pull request is merged to master all the ci is run again. I don't know how much this is needed since we run all the ci on pull requests and dont merge if something doesn't pass.